context/gsm_build.go

```diff
@@ -25,7 +26,7 @@ func BuildGSMPDUSessionEstablishmentAccept(smContext *SMContext) ([]byte, error)
 	pDUSessionEstablishmentAccept.SetPDUSessionID(uint8(smContext.PDUSessionID))
 	pDUSessionEstablishmentAccept.SetMessageType(nas.MsgTypePDUSessionEstablishmentAccept)
 	pDUSessionEstablishmentAccept.SetExtendedProtocolDiscriminator(nasMessage.Epd5GSSessionManagementMessage)
-	pDUSessionEstablishmentAccept.SetPTI(0x00)
+	pDUSessionEstablishmentAccept.SetPTI(smContext.Pti)
```

Comment:
Hi, @edingroot 
According to TS24.501 6.2.1
The PTI value should be set in **every**  response from UE-requested transaction.
e.g., PTI should also be set in `BuildGSMPDUSessionEstablishmentReject`, `BuildGSMPDUSessionModificationCommand` ... so on and so forth.